contrib/current-cgroup

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ "$#" -gt 0 ] && { [ "$1" = "-h" ] || [ "$1" = "--help" ] ; }; then
+  echo "Helper script to find the current cgroup v2 path"
+  echo "Usage: [PID=PROCESS_ID] $0"
+  echo "  If PID is specified as environment variable, the cgroup path of"
+  echo "  the given PID will be queried. Otherwise the PID of the script"
+  echo "  process itself will be used."
+  exit 1
+fi
+
+PID="${PID:-$$}"
+
+if [ -e /sys/fs/cgroup/unified ]; then
```

Comment:
Maybe this could be checked differently.

To find the cgroup2 mountpoint:
```
$ mount -l -t cgroup2|awk '{print $3}'
/sys/fs/cgroup/unified
```

To check if a mountpoint is cgroup2:
```
$ stat --file-system --format %T /sys/fs/cgroup/unified/
cgroup2fs
```

Reply:
Good, I changed it and used the `stat` to check for tmpfs in the benchmark test load.